java/daikon/chicory/Runtime.java

```diff
@@ -146,43 +149,6 @@ private Runtime()
         throw new Error("Do not create instances of Runtime");
     }
 
-    /** Printf to dtrace file. **/
```

Comment:
What is the reason for deleting these methods?
I guess it's because they are dead code?